campaigns/csp/guides/bandwidth10dlcCampaignCspApiGuide.md

```diff
@@ -0,0 +1,875 @@
+            <AutoRenewal>true</AutoRenewal>
+            <BilledDate>2021-05-25T00:00</BilledDate>
+            <BrandId>B9AVERT</BrandId>
+            <Vertical>HEALTHCARE</Vertical>
+            <Usecase>2FA</Usecase>
+            <SubUsecases/>
+            <Description>test</Description>
+            <ResellerId>RPTAYWZ</ResellerId>
+            <EmbeddedLink>false</EmbeddedLink>
+            <EmbeddedPhone>false</EmbeddedPhone>
+            <NumberPool>false</NumberPool>
+            <AgeGated>false</AgeGated>
+            <DirectLending>false</DirectLending>
+            <SubscriberOptIn>true</SubscriberOptIn>
+            <SubscriberOptOut>false</SubscriberOptOut>
+            <SubscriberHelp>true</SubscriberHelp>
+            <Sample1>test</Sample1>
+            <AffiliateMarketing>false</AffiliateMarketing>
+            <MnoStatusList>
+                <MnoStatus>
+                    <MnoName>ATT</MnoName>
+                    <MnoId>10017</MnoId>
+                    <Status>APPROVED</Status>
+                </MnoStatus>
+                <MnoStatus>
+                    <MnoName>TMO</MnoName>
+                    <MnoId>10035</MnoId>
+                    <Status>APPROVED</Status>
+                </MnoStatus>
+            </MnoStatusList>
+            <SharingStatus>true</SharingStatus>
+        </Campaign>
+        <Campaign>
+            <CampaignId>CU3UTYW</CampaignId>
+            <Status>EXPIRED</Status>
+            <CreateDate>2021-06-03T11:50:16</CreateDate>
+            <AutoRenewal>false</AutoRenewal>
+            <BilledDate>2021-06-03T00:00</BilledDate>
+            <BrandId>B9AVERT</BrandId>
+            <Vertical>ENERGY</Vertical>
+            <Usecase>2FA</Usecase>
+            <SubUsecases/>
+            <Description>test</Description>
+            <ResellerId>RPTAYWZ</ResellerId>
+            <EmbeddedLink>false</EmbeddedLink>
+            <EmbeddedPhone>false</EmbeddedPhone>
+            <NumberPool>false</NumberPool>
+            <AgeGated>false</AgeGated>
+            <DirectLending>false</DirectLending>
+            <SubscriberOptIn>true</SubscriberOptIn>
+            <SubscriberOptOut>false</SubscriberOptOut>
+            <SubscriberHelp>true</SubscriberHelp>
+            <Sample1>test</Sample1>
+            <Sample2>test 15</Sample2>
+            <AffiliateMarketing>false</AffiliateMarketing>
+            <MnoStatusList>
+                <MnoStatus>
+                    <MnoName>ATT</MnoName>
+                    <MnoId>10017</MnoId>
+                    <Status>APPROVED</Status>
+                </MnoStatus>
+                <MnoStatus>
+                    <MnoName>TMO</MnoName>
+                    <MnoId>10035</MnoId>
+                    <Status>APPROVED</Status>
+                </MnoStatus>
+            </MnoStatusList>
+            <SharingStatus>true</SharingStatus>
+        </Campaign>
+    </Campaigns>
+    <TotalCount>17</TotalCount>
+</CampaignsResponse>
+```
+
+### Error Response
```

Comment:
Is this response different if the CampaignManagement feature is enabled on the account but the user does not have the appropriate role?

Reply:
Yeah will just have different error message